### PR TITLE
Update template_extras logic in get_minutes 

### LINKED ISSUE #187: Missing some previous events
The events listing is missing the meeting of 6/22/2017: https://boardagendas.metro.net/events/

The data appears to be in the database. When I run [this code from the django shell](https://github.com/datamade/la-metro-councilmatic/blob/master/lametro/views.py#L254-L261) I get what I expect.

(on the server
```bash 
cd /home/datamade/lametro
workon lametro
python manage.py shell
```
```python
>>> from councilmatic_core.models import Event
>>> import datetime
>>> day_grouper = lambda x: (x.start_time.year, x.start_time.month, x.start_time.day)
>>>  past_events = Event.objects.filter(start_time__lt=datetime.datetime.now()).order_by('-start_time')
>>> [(datetime.date(*a),sorted(b, key=lambda x: x.start_time)) for a,b in itertools.groupby(past_events, key=day_grouper)]
[(datetime.date(2017, 7, 20), [<Event: Event object>, <Event: Event object>, <Event: Event object>, <Event: Event object>]), 
 (datetime.date(2017, 7, 19), [<Event: Event object>, <Event: Event object>, <Event: Event object>]), 
 (datetime.date(2017, 6, 22), [<Event: Event object>, <Event: Event object>]), 
... ]
```

I don't see anything in the template code that would filter out the 6/22 event:

https://github.com/datamade/la-metro-councilmatic/blob/master/lametro/templates/lametro/events.html#L82-L86

and https://github.com/datamade/la-metro-councilmatic/blob/master/lametro/templates/partials/past_event_day.html

### PR DESCRIPTION
**Pull request by reginafcompton**
This pull request closes #187

### CODE REVIEW

File: lametro/templatetags/lametro_extras.py [hunk 1]
```diff
@@ -125,10 +125,9 @@ def get_minutes(event_id):
         sqs = SearchQuerySet().filter(content=content).all()
         if sqs:
             for q in sqs:
-                if q.object.bill_type == 'Minutes' and q.object.slug:
-
-                    if re.search(content, q.object.ocr_full_text, re.IGNORECASE):
-                        return '/board-report/' + q.object.slug
+                if q.object.bill_type == 'Minutes' and q.object.slug and q.object.ocr_full_text:
```
Comment (fgregg) [inline lametro/templatetags/lametro_extras.py]: cool. For readability, let's split these long lines

```python
if (q.object.bill_type == 'Minutes' and
    q.objects.slug and
    q.object.ocr_full_text):
    ....

**Comment by reginafcompton** [comment]
@fgregg: changes made for readability - let's plan to deploy in the morning, if this looks okay.

**Comment by fgregg** [comment]
Looks good. Tag Shelly here and let her know we have a fix.

File: lametro/templatetags/lametro_extras.py [hunk 2]
```diff
@@ -125,10 +125,11 @@ def get_minutes(event_id):
...
         if sqs:
             for q in sqs:
-                if q.object.bill_type == 'Minutes' and q.object.slug:
-
-                    if re.search(content, q.object.ocr_full_text, re.IGNORECASE):
-                        return '/board-report/' + q.object.slug
+                if (q.object.bill_type == 'Minutes' and 
+                    q.object.slug and 
+                    q.object.ocr_full_text):
+                        if re.search(content, q.object.ocr_full_text, re.IGNORECASE):
```
Comment (fgregg) [inline lametro/templatetags/lametro_extras.py]: This looks too far indented.

Comment (reginafcompton) [inline lametro/templatetags/lametro_extras.py]: @fgregg I do not see an indentation problem, when I expand the code and place this snippet in context. Can you say more about what you expect?

**Pull request merged**